android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -67,15 +69,11 @@ private static int getGoogleApiKeyResourceId(Context context){
                                                 context.getPackageName());
   }
 
-  private static String getApiKey(Context context){
-    int resourceId = getGoogleApiKeyResourceId(context);
-    return resourceId != 0 ? context.getString(resourceId) : "";
-  }
-
   private static String getCurrentPwsEndpoint(Context context){
     String defaultEndpoint = isGoogleApiKeyAvailable(context) ?
-      convertToSettingsEndpointFormat(GOOGLE_ENDPOINT, GOOGLE_ENDPOINT_VERSION) :
-      convertToSettingsEndpointFormat(PROD_ENDPOINT, PROD_ENDPOINT_VERSION);
+      convertToSettingsEndpointFormat(GOOGLE_ENDPOINT, GOOGLE_ENDPOINT_VERSION,
```

Comment:
This line is pretty verbose, could you switch to an if/else instead for clarity?